app/controllers/car_controller.rb

```diff
@@ -0,0 +1,13 @@
+class CarController < ApplicationController
+  def new
+    @user = current_user
+  end
+
+  def create
+    # TODO
+    # before redirect, send params to facade/service to make a graphql mutation query
+    # to save car into backend db
+    # respond with new car data
```

Comment:
Can we build this out with a mocked out spec that takes the query and returns the expected data?

Reply:
This lays the ground work, but story #18 should build on this functionality. This is an example where we didn't do a great job with grooming the stories beforehand.